code/drasil-example/template/lib/Drasil/Template/Body.hs

```diff
@@ -9,6 +9,8 @@ import Language.Drasil
 import Drasil.SRSDocument
 import Theory.Drasil (DataDefinition, GenDefn, InstanceModel, TheoryModel)
 import qualified Language.Drasil.Sentence.Combinators as S
+import Data.Drasil.Concepts.Documentation(doccon, doccon')
```

Comment:
I would add a space here for consistency.
```suggestion
import Data.Drasil.Concepts.Documentation (doccon, doccon')
```

Reply:
I will change that right now!